aws-android-sdk-iot/src/test/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -864,7 +864,7 @@ public void onMessageArrived(String topic, byte[] data) {
      */
     @Ignore
     @Test
-    public void mqttPersistantSession() throws Exception {
+    public void mqttPersistentSession() throws Exception {
```

Comment:
can you add a unit test for setting username?

Reply:
Yeah. I am working on it. I will update the PR when it is ready

Reply:
@desokroshan added the tests